tests/test_postgresql.py

```diff
@@ -347,6 +347,7 @@ def test_write_postgresql_and_sanitize_auto_conf(self):
     def test_follow(self):
         self.p.call_nowait(CallbackAction.ON_START)
         m = RemoteMember('1', {'restore_command': '2', 'primary_slot_name': 'foo', 'conn_kwargs': {'host': 'bar'}})
+        self.assertEqual(m.disallowed_attr, None)
```

Comment:
I wonder if declaring `RemoteMember.__getattr__()` as returning Optional[Any] would work better?

Reply:
Do you mean in the test or in the code?

The extra `return None` is not part of typing tests. It comes from https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/inconsistent-return-statements.html

Reply:
I mean that `return None` is redundant, it is returned implicitly.

Reply:
Or

```
    def __getattr__(self, name: str) -> Any:
        """Dictionary style key lookup.

        :param name: key to lookup.

        :returns: value of *name* key in :attr:`~RemoteMember.data` if key *name* is in
                  :cvar:`~RemoteMember.ALLOWED_KEYS`, else ``None``.
        """
        return self.data.get(name) if name in RemoteMember.ALLOWED_KEYS else None
```

Reply:
Removed both `return None` and test.